### PR TITLE
Update docs for using video component

### PR DESCRIPTION
**Pull request by Krasavinigor**
**Issue**: Outdated information of supported components https://github.com/microsoft/react-native-code-push/issues/1879

**Solution**: Update documents

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,7 +100,11 @@ The following list represents the set of components (and props) that don't curre
 
 As new core components are released, which support referencing assets, we'll update this list to ensure users know what exactly they can expect to update using CodePush.
 
+*Note: CodePush works correctly with Video component only if using `require`. Example:*
```
Comment (district-michael) [inline README.md:103]: I'd reword this slightly to:
*Note: CodePush only works with Video components when using `require` in the source prop. For example:*

File: README.md [hunk 2]
```diff
@@ -99,7 +100,11 @@ The following list represents the set of components (and props) that don't curre
...
 
+```javascript
+<Video source={require("foo")} />
```
Comment (district-michael) [inline README.md:106]: I'd also change the example here to actually include the file extension, ie `require("./foo.mp4")`

File: README.md (hunk 1 above)
Comment (Krasavinigor) [reply README.md]: Thank you!

File: README.md (hunk 2 above)
Comment (Krasavinigor) [reply README.md]: Yes, it will be better!
